src/resources/touch_esp32.cc

```diff
@@ -274,13 +272,12 @@ PRIMITIVE(read) {
   ARGS(IntResource, resource);
   touch_pad_t pad = static_cast<touch_pad_t>(resource->id());
 
-#if CONFIG_IDF_TARGET_ESP32S3 || CONFIG_IDF_TARGET_ESP32S2
-  uint32_t val;
-  esp_err_t err = touch_pad_read_raw_data(pad, &val);
-#else
+#ifdef CONFIG_IDF_TARGET_ESP32
   uint16_t val;
-  esp_err_t err = touch_pad_read(pad, &val);
+#else
+  uint32_t val;
 #endif
+  esp_err_t err = touch_pad_read_raw_data(pad, &val);
```

Comment:
Is the change to always use raw okay?

Reply:
Yes. It just avoids the filter. Should be ok.

Reply:
SG!